log_shipping/clent_log_shipping_s3_bucket_create.go

```diff
@@ -0,0 +1,81 @@
+package log_shipping
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	logzio_client "github.com/logzio/logzio_terraform_client"
+)
+
+const createS3BucketServiceMethod string = http.MethodPost
```

Comment:
Change to a multiple const format. For instance, https://github.com/logzio/logzio_terraform_client/blob/master/endpoints/client_endpoints.go#L17)